include/swift/AST/PrintOptions.h

```diff
@@ -425,6 +425,7 @@ struct PrintOptions {
     result.AbstractAccessors = false;
     result.PrintForSIL = true;
     result.PrintInSILBody = true;
+    result.PreferTypeRepr = false;
```

Comment:
Why are you changing `printSIL`, `Decl::print` and swift-ide-test's Type reconstruction to use `PreferTypeRepr = false`?